.github/ISSUE_TEMPLATE/new-sample-setup.md

```diff
@@ -9,15 +9,14 @@ assignees: ''
 
 ## New Sample Checklist
 
-<!-- markdown-link-check-disable-next-line -->
-Ensure you've read the [Contributing Guide (Microsoft Internal only.)](https://dev.azure.com/csedtd/_git/DataOps?path=%2FCONTRIBUTING.md&_a=preview)
+Ensure you've read the [Contributing Guide](/CONTRIBUTING.md).
 
 ## Tooling
 
 - [ ] Setup/Configure Azure Subscriptions (if applicable)
 - [ ] CI/CD Tooling (Azure DevOps)
-- [ ] Setup Work Item tracking Tooling (GitHub Project)
-- [ ] Collaboration and Communication Tooling  (Teams)
+- [ ] Setup Work Item tracking Tooling (i.e., GitHub Projects - Microsoft Employees only)
```

Comment:
Should there be specific steps for MS employees if this is to be open-sourced?

Reply:
For MS employee specific detailed guidance (MS policy, ISE specific guidance, etc), we should document that in a separate internal-only guide.